### DISCUSSION TITLE
Battery charge level

### DISCUSSION THREAD

**Discussion by christianpele** [hultenvp/solis-sensor#267] [Q&A]
Hi,
I have been looking around how to show the battery charge level in home assistant. On the Solis app you got a percentage
Is this missing from the API or is there a formula for?
Hope someone can help me through this
Thx
Christian

![IMG_12CBA58483C4-1 2](https://user-images.githubusercontent.com/83245416/226133564-cb724b5e-f97e-4648-9e39-8c8c036d60a2.jpeg)

**Comment by alienatedsec** [comment]
@christianpele `sensor.[yourgivenname]_remaining_battery_capacity`

**Reply by christianpele** [reply]
Thnaks a lot!! Whaaauuww can't believe I missed that sensor.
